config/api.yml

```diff
@@ -724,6 +737,13 @@
     - :subcollection
     :methods: *70174834085620
     :klass: Tag
+    :collection_actions:
+      :post:
+        - :name: add
+    :resource_actions:
+      :post:
+        - :name: edit
+        - :name: delete
```

Comment:
Ah, I remember needing to ask about this but must have forgot. Any clues as to the right identifiers to use? Couldn't find anything in miq_product_features.yml.

/cc @gtanzillo